rancher/kubernetes/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Kubernets in Rancher
+layout: rancher-default
+---
+
+## Kubernetes
+---
+
+To deploy Kubernetes in Rancher, you'll first need to create a new [environment]({{site.baseurl}}/rancher/configuration/environments/) that has specified the cluster management to be Kubernetes. 
+
+### Creating a Kubernetes Environment
+
+In the dropdown of environments, click on the **Manage Environments**. A list of existing environments will be displayed. The _Default_ environment that is started when you launch Rancher is based on the _Cattle_ cluster management. You can choose to change the cluster management type of the _Default_ environment or  create a new environment that uses _Kubernetes_ as the cluster management.
+
+To create a new environment, click on **Add Environment**, select **Kubernetes** as the cluster management, provide a **Name**, **Description** (Optional). If [access control]({{site.baseurl}}/rancher/configuration/access-control/) is turned on, you can [add members]({{site.baseurl}}/rancher/configuration/environments/#editing-members) and select their [membership role]({{site.baseurl}}/rancher/configuration/environments/#membership-roles). Anyone added to the membership list would have access to your environment. 
+
+If you choose to change an existing environment's cluster management, click on **Edit** in the environment's dropdown. Change the cluster management to _Kubernetes_ and click on **Save**. 
```

Comment:
Same thing, if we change container orchestration but services are already running, what happens?